### PR TITLE
Checkstyle warnings removed

### PR DESCRIPTION
**Pull request by AndreasMager**

### CODE REVIEW

File: broker/src/main/java/io/moquette/server/netty/NettyMQTTHandler.java [hunk 1]
```diff
@@ -83,6 +79,8 @@ public void channelRead(ChannelHandlerContext ctx, Object message) {
                     MqttMessage pingResp = new MqttMessage(pingHeader);
                     ctx.writeAndFlush(pingResp);
                     break;
+                default:
```
Comment (andsel) [inline broker/src/main/java/io/moquette/server/netty/NettyMQTTHandler.java:82]: is it really needed?

Comment (AndreasMager) [reply broker/src/main/java/io/moquette/server/netty/NettyMQTTHandler.java:82]: Its a default warning in eclipse.

Comment (AndreasMager) [reply broker/src/main/java/io/moquette/server/netty/NettyMQTTHandler.java:82]: I will add an Error logger to the default case.

**Comment by andsel** [comment]
At the end of the fixes requested, please squash all commits, to be ready to merge

**Comment by AndreasMager** [comment]
Ready

**Comment by andsel** [comment]
Thanks a lot Andreas!!

**Pull request merged**